vllm_ascend/ops/common_fused_moe.py

```diff
@@ -33,7 +34,15 @@ def unquantized_fused_moe_init_func(self, *args, **kwargs):
     original_unquantized_fused_moe_init_func(self, *args, **kwargs)
     vllm_config = get_current_vllm_config()
     self.max_num_batched_tokens = vllm_config.scheduler_config.max_num_batched_tokens
-    self.use_aclgraph = vllm_config.compilation_config.level == CompilationLevel.PIECEWISE and not vllm_config.model_config.enforce_eager
+
+    ascend_config = get_ascend_config()
```

Comment:
I suggest referring to #1841 and adding a new class called `AscendUnquantizedFusedMoEMethod`, incorporating the ACLGraph switch into this class's `__init__` method, while also placing the new `forward_oot` method within this class.

Reply:
@ApsarasX it's on the todo list. we'll first combine common_fused_moe and fused_moe into one. Then refactor to Custom OP register way.